src/ProjectService.Models.Dto/Requests/CreateImageRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.ProjectService.Models.Dto.Enums;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Requests
+{
+    public class CreateImageRequest
+    {
+        public Guid Id { get; set; }
```

Comment:
If you write "Id" it means the entity id => in this case CreateImageRequestId. What do you mean when writing id? rewrite class to record

Reply:
done